plutus-ledger-api/src/PlutusLedgerApi/Common/Eval.hs

```diff
@@ -132,6 +133,25 @@ mkDynEvaluationContext ver newCMP =
 assertWellFormedCostModelParams :: MonadError CostModelApplyError m => Plutus.CostModelParams -> m ()
 assertWellFormedCostModelParams = void . Plutus.applyCostModelParams Plutus.defaultCekCostModel
 
+evaluateTerm
```

Comment:
```suggestion
-- | Evaluate a fully-applied term using the CEK machine. Useful for mimicing the behaviour of the on-chain evaluator.
evaluateTerm
```

Reply:
@michaelpj @effectfully What do you mean by "mimicking the behaviour of the on-chain evaluator"? This function `evaluateTerm` is what the actual on-chain node calls through the `plutus-ledger-api` !?

Reply:
But it's not _the actual function_ that the ledger calls. So we are still trying to make sure we mimic it's behaviour, and it could go wrong if there was something important in the bit we're not calling.